api/tests/views_tests.py

```diff
@@ -39,6 +40,7 @@ def get_user(client: APIClient) -> User:
 
 
 @pytest.mark.parametrize("format", ("yaml", "json"))
+@override_settings(API_DOCS_ENABLED=True)
```

Comment:
Seems like the [test is failing](https://app.circleci.com/pipelines/github/mozilla/fx-private-relay/9848/workflows/cde4aef2-a439-4851-811b-f6051d9c5208/jobs/55033) because the url is not registered when the test started... This may be fixed if we update the logic for `API_DOCS_ENABLED` in privaterelay/settings.py (see comment below).

Reply:
Cool, thanks. Giving that a try.